examples/snippets/interleaved-tables-before-7.1/models/album.rb

```diff
@@ -0,0 +1,20 @@
+# Copyright 2022 Google LLC
```

Comment:
2023?

Reply:
This is the existing sample for using the third-party composite primary key gem. GitHub is just not very good at showing a change that renames a folder.